Backend/routes/listings.js

```diff
@@ -41,7 +41,9 @@ router.post(
     } = req.body;
 
     const price =
-      isAuction === "true" ? parseFloat(initialBid) : parseFloat(req.body.price);
+      isAuction === "true"
```

Comment:
let's have isAuction === "true" be its own variable so you don't have to type it everywhere